src/benchmark/CMakeLists.txt

```diff
@@ -30,6 +30,8 @@ target_link_libraries(
     benchmark
 )
 
+target_link_libraries_system(hyriseMicroBenchmarks nlohmann_json::nlohmann_json)
```

Comment:
Is there a more elegant solution to this problem?

Reply:
... which is?